test/e2e/flowaggregator_test.go

```diff
@@ -1815,6 +1837,35 @@ func createToExternalTestServer(t *testing.T, data *TestData) *PodIPs {
 	return serverIPs
 }
 
+func getAndCheckFlowAggregatorMetrics(t *testing.T, data *TestData) error {
+	flowAggPod, err := data.getFlowAggregator()
+	if err != nil {
+		return fmt.Errorf("error when getting flow-aggregator Pod: %v", err)
+	}
+	podName := flowAggPod.Name
+	command := []string{"antctl", "get", "recordmetrics", "-o", "json"}
+	if err := wait.Poll(defaultInterval, 2*defaultTimeout, func() (bool, error) {
+		stdout, _, err := runAntctl(podName, command, data)
+		if err != nil {
+			t.Logf("error when requesting recordmetrics, %v", err)
+			return false, nil
+		}
+		metrics := &recordmetrics.Response{}
+		if err := json.Unmarshal([]byte(stdout), metrics); err != nil {
+			t.Fatalf("error when decoding recordmetrics, %v", err)
+			return false, nil
+		}
+		if metrics.NumConnToCollector != int64(clusterInfo.numNodes) || !metrics.WithClickHouseExporter || !metrics.WithIPFIXExporter || metrics.NumRecordsExported == 0 {
+			t.Logf("metrics are not correct. Current metrics: NumConnToCollector=%d, ClickHouseExporter=%v, IPFIXExporter=%v, NumRecordsExported=%d", metrics.NumConnToCollector, metrics.WithClickHouseExporter, metrics.WithIPFIXExporter, metrics.NumRecordsExported)
+			return false, nil
+		}
+		return true, nil
+	}); err != nil {
+		return fmt.Errorf("error when checking recordmetrics for Flow Aggregator: %v", err)
```

Comment:
nit: let's try to always wrap errors with `%w` instead of `%v`

Reply:
Fixed. Thanks.